library-udf/src/main/java/org/apache/iotdb/library/dprofile/util/MADSketch.java

```diff
@@ -262,8 +265,9 @@ public double[] getValid_range() {
 
   public void show(Bucket[] buckets) {
     for (Bucket bucket : buckets) {
-      Logger logger = Logger.getLogger(getClass());
-      logger.debug(bucket.index + ": " + bucket.count);
+      if (logger.isDebugEnabled()) {
+        logger.debug(bucket.index + ": " + bucket.count);
+      }
```

Comment:
if (logger.isDebugEnabled()) {
  for (Bucket bucket : buckets) {
    logger.debug(bucket.index + ": " + bucket.count);
  }
}